docs/Setup/google.md

```diff
@@ -1,5 +1,146 @@
 # 🌐 Google OAuth2 🔑 Guide
 
-```note
-Hi! Soon, Here will be the docs 📝 for Google provider!
-```
\ No newline at end of file
+The `Google` provider lets you sign in 🔓 users using their Google accounts through OAuth2.
+
+---
+
+## 🔧 Google App Setup Guide
+
+To use Google OAuth2 in your Django REST Framework app, you need to set up a Google App. Here's a step-by-step guide to obtain the necessary credentials (`client_id`, `client_secret`, `redirect_uri`) and configure your app.
```

Comment:
google.md file faqat google ga tegishli bo'lgan cridentiallarni olish va google Oauth2 app ni qanaqa qilib setup qilish haqida bo'lish kerak. 

```suggestion
To use GitHub OAuth2 in your app, you need to set up a GitHub OAuth2 App. Here's a step-by-step guide to obtain the necessary credentials (`client_id`, `client_secret`, `redirect_uri`) and configure your app.
```

GitHub docs ga qarab ko'rsez bo'ladi: https://omni-libraries.mukhsin.space/setup/github


Change the rest of the docs according to this review comment please.